frontend/src/components/ChoicesFilterSection.tsx

```diff
@@ -35,6 +60,7 @@ const ChoicesFilterSection = ({ title, value, choices, onChange }: Props) => {
                   onChange={handleChange}
                   name={choiceValue}
                   size="small"
+                  data-testid={title + ': ' + choiceLabel}
```

Comment:
Commenting about line 57: `icon={<CheckCircleOutline />}`. Could you change the icon to a square when it is possible to select multiple? (I believe that is the usual UI for this type of multiple choice form)